### PR TITLE
Update website for Collector v0.27.0

### PR DESCRIPTION
**Pull request by humivo**
Add blog posts and download contents page.

### CODE REVIEW

File: src/content/BlogPosts/blogPosts.yaml [hunk 1]
```diff
@@ -39,7 +47,7 @@ blogs:
     author: "Bryan Aguilar"
     date: "02-Feb-2023"
     body:
-      "AWS Distro for OpenTelemetry v0.25.0 is now available. You can download the latest AWS Distro for OpenTelemetry Collector image
```
Comment (bryan-aguilar) [inline src/content/BlogPosts/blogPosts.yaml:42]: good catch

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.27.0.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: 'AWS Distro for OpenTelemetry v0.27.0'
+description:
+    This blog post is the release announcement for ADOT v0.27.0
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.27.0.mdx:4]:
```suggestion
    This blog post is the release announcement for ADOT Collector v0.27.0
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.27.0.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+import SectionSeparator from "components/MdxSectionSeparator/sectionSeparator.jsx"
+
+<SectionSeparator />
+
+[AWS Distro for OpenTelemetry (ADOT)](https://aws-otel.github.io/) v0.27.0 is now available.
```
Comment (bryan-aguilar) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.27.0.mdx:11]:
```suggestion
[AWS Distro for OpenTelemetry (ADOT)](https://aws-otel.github.io/) Collector v0.27.0 is now available.
```

**Review by bryan-aguilar — APPROVED** [review]

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.27.0.mdx (hunk 2 above)
Comment (humivo) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.27.0.mdx]: Added

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.27.0.mdx (hunk 3 above)
Comment (humivo) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-collector-v0.27.0.mdx]: Added

**Pull request merged**